.prettierrc.json

```diff
@@ -1,15 +1,17 @@
 {
-    "endOfLine":"auto",
-    "useTabs": false,
-    "tabWidth": 4,
-    "singleQuote": true,
-    "trailingComma": "none",
-    "printWidth": 80,
-    "htmlWhitespaceSensitivity": "ignore",
-    "overrides": [
-        {
-            "files": "*.html",
-            "options": { "parser": "lwc" }
-        }
-    ]
-}
+  "endOfLine": "auto",
+  "useTabs": false,
+  "tabWidth": 4,
+  "singleQuote": true,
+  "trailingComma": "none",
+  "printWidth": 120,
+  "htmlWhitespaceSensitivity": "ignore",
+  "overrides": [
+    {
+      "files": "*.html",
+      "options": {
+        "parser": "lwc"
+      }
+    }
+  ]
+}
```

Comment:
Also [this](https://github.com/salesforcecli/plugin-lightning-dev/blob/main/.prettierrc.json)?

Reply:
Same thing... extending from`@salesforce/prettier-config` gives very basic configuration which is not adequate